### PR TITLE
Fix tests vof pw

### PR DESCRIPTION
**Pull request by jeannejoachim**
# Description of the problem

- two tests using the peeling/wetting mechanisms where broken on the CI

# Description of the solution

- found the problem with valgrind: the `n_q_point` where not at the correct value
- changed and made this more explicit

# How Has This Been Tested?

- 2 tests for pw retested on my machine and with valgrind. Hope the CI passes now!

### CODE REVIEW

**Comment by jeannejoachim** [comment]
This fixes the issue #483

File: source/core/parameters.cc [hunk 1]
```diff
@@ -1987,7 +1987,7 @@ namespace Parameters
         prm.set("Function expression", "0; 0; 0");
       prm.leave_subsection();
 
-      unsigned int max_ib_particles = 1000;
+      unsigned int max_ib_particles = 10;
```
Comment (hepap) [inline source/core/parameters.cc:1990]: Is the `max_ib_particles = 10` still right?

**Review by hepap — APPROVED** [review]
Looks good to me! Vive Valgrind! :1st_place_medal:

File: source/core/parameters.cc (hunk 1 above)
Comment (jeannejoachim) [reply source/core/parameters.cc:1990]: yes this is the value on master now!

**Review by lpsaavedra — APPROVED** [review]
Everything looks good to me!

**Review by blaisb — APPROVED** [review]

**Pull request merged**